HomeWork05/optional_task3.py

```diff
@@ -0,0 +1,7 @@
+"""Скажу чесно: код вкрав з першого посилання гуглу, бо я лютий гуманітарій і мені дуже важко зрозуміти такі штуки,
```

Comment:
Yeah, it is really hard to understand loops in loops at the beginning. But anyway it is great that you have tried